lib/ansible/modules/yum_repository.py

```diff
@@ -171,6 +171,7 @@
       - This tells yum whether or not HTTP/1.1 keepalive should be used with
         this repository. This can improve transfer speeds by using one
         connection when downloading multiple files from a repository.
+      - This option is only for Yum, not DNF. It has been removed in RHEL 8.
```

Comment:
We should note that it holds for YUM version 3 and below.

Reply:
Thanks, I've updated it. "below" is ancient so I just said Yum 3.